src/Libraries/GeometryUIWpf/PanelSurfaceBoundaryConditions.cs

```diff
@@ -0,0 +1,53 @@
+using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Runtime;
+using CoreNodeModels;
+using DSCore;
+using Dynamo.Graph.Nodes;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+
+namespace GeometryUIWpf
+{
+    [IsVisibleInDynamoLibrary(true)]
+    [NodeName("PanelSurfaceBoundaryConditions")]
+    [NodeCategory("Geometry.PanelSurface")]
```

Comment:
nodeCategory and OutputNames do no seem to take effect ?
any ideas @aparajit-pratap  ?

Reply:
I think you need to modify the layoutspec.json. See the `exporttosat `node as an example:
![image](https://github.com/DynamoDS/Dynamo/assets/5710686/bd4dc7e1-4da3-4247-9b26-7b4b5a3fa46a)